prime-angular-frontend/src/app/shared/components/document-upload/document-upload/document-upload.component.ts

```diff
@@ -78,6 +78,23 @@ export class DocumentUploadComponent implements OnInit {
     this.jwt = await this.keycloakTokenService.token();
   }
 
+  private getIdleText(allowedFileTypesMap: { [key: string]: string }): string {
+    const baseText = 'Click to Browse or Drop files here.';
+    if (allowedFileTypesMap == null) {
+      return baseText;
+    }
+
+    const types = Object.values(allowedFileTypesMap);
+    let typeText = types.pop();
+
+    if (types.length > 0) {
+      const allButLast = types.join(', ');
+      typeText = `${allButLast} or ${typeText}`;
+    }
+
+    return `${baseText} Files must be ${typeText}`;
```

Comment:
To follow punctuation should we throw a '.' at the end.

Reply:
I was thinking about that too. if we did, the label would read (for example)

`Click to Browse or Drop files here. Files must be .pdf.`

that second period looks a little off

Reply:
But I just noticed that the label didn't have a period at the end before, so im gonna remove that from the base label.